web/styles/settings.css

```diff
@@ -1316,9 +1316,7 @@ label.preferences-radio-choice-label {
 
             @media (width >= $md_min) {
                 .tab-switcher {
-                    position: absolute;
                     display: flex;
-                    left: 0.7142em; /* 10px at 14px em */
                     z-index: 1;
```

Comment:
With positioning gone, do we still have need for the `z-index` value here?

Reply:
Seems like it can be removed, yeah, thanks!